go-controller/pkg/config/cni.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 var ErrorAttachDefNotOvnManaged = errors.New("net-attach-def not managed by OVN")
+var ErrorPluginsListSize = errors.New("Plugins list size must equal one")
```

Comment:
I'd rather have something with more semantic sense. 

Like `CNI plugin chaining is not supported`. 

Probably have a different error for when the user provisions a configuration without plugins defined in the list ? (... I agree your current proposal has the benefit of introducing a single error type).


Let's hear from a maintainer before we act on this.